src/frontend/src/binder/relation/mod.rs

```diff
@@ -438,7 +438,10 @@ impl Binder {
                 for_system_time_as_of_proctime,
             } => self.bind_relation_by_name(name, alias, for_system_time_as_of_proctime),
             TableFactor::TableFunction { name, alias, args } => {
-                self.bind_table_function(name, alias, args)
+                self.try_mark_lateral_as_visible();
```

Comment:
Do you know why it's named as `try_...`? 🤣

Reply:
Maybe it needs to check if `lateral_context` exists.
```rust
    fn try_mark_lateral_as_visible(&mut self) {
        if let Some(mut ctx) = self.lateral_contexts.pop() {
            ctx.is_visible = true;
            self.lateral_contexts.push(ctx);
        }
    }
```